### DISCUSSION TITLE
Part of productive build

### DISCUSSION THREAD

**Discussion by mackerl** [quarkiverse/quarkus-quinoa#645] [General]
Hi,
I maybe missed something, but I could not figure out if or how quinoa can be used in production? 

I am having an angular frontend within the src/main/webui folder which runs fine in quarkus dev environment. However the jars that come out of quarkus build does not seem to have any parts of the npm build included.
Do I still need to trigger the npm build and copy the dist files to META-INF/resources? I thought this is not necessary when using quinoa.

**Comment by melloware** [comment]
Nope that should all happen automatically.   if you are using Angular 17 see this thread: https://github.com/quarkiverse/quarkus-quinoa/issues/579
